packages/itwinui-react/src/core/Table/cells/DefaultCell.tsx

```diff
@@ -86,18 +94,22 @@ export const DefaultCell = <T extends Record<string, unknown>>(
     >
       <ShadowRoot key={`${cellElementKey}-shadow-root`} flush={false} css={css}>
         <slot name='start' />
-        <div
-          className='_iui-table-cell-main-content'
-          onClick={(e) => e.stopPropagation()}
+
+        <TableCellContent
+          isCustomCell={isCustomCell}
+          isCustomRenderer={isCustomRenderer}
```

Comment:
nit: These two can be combined into a single prop. `TableCellContent` does not care about their individual values.

Reply:
Not outdated because the `isCustomRenderer` prop is still mixing concerns from the parent component. Clearer name would be `shouldRenderWrapper` or something similar. The value passed to this prop may change over time.